geode-pulse/src/main/webapp/WEB-INF/web.xml

```diff
@@ -32,7 +32,7 @@
   </servlet>
   <servlet-mapping>
     <servlet-name>mvc-dispatcher</servlet-name>
-    <url-pattern>/pulse/*</url-pattern>
```

Comment:
This seems dangerous. Is this ok because we are already deploying the war under /pulse? Have you tested starting pulse and connect to gfsh using http?